### PR TITLE
UCM: set event handler wo installing hooks

### PR DESCRIPTION
**Pull request by brminich**

### CODE REVIEW

File: src/ucm/api/ucm.h [hunk 1]
```diff
@@ -26,16 +26,20 @@ BEGIN_C_DECLS
...
+    UCM_EVENT_SBRK           = UCS_BIT(5),
 
     /* Aggregate events */
-    UCM_EVENT_VM_MAPPED   = UCS_BIT(16),
-    UCM_EVENT_VM_UNMAPPED = UCS_BIT(17)
+    UCM_EVENT_VM_MAPPED      = UCS_BIT(16),
+    UCM_EVENT_VM_UNMAPPED    = UCS_BIT(17),
+
+    /* Auxiliary flags */
+    UCM_EVENT_AUX_NO_INSTALL = UCS_BIT(24)
```
Comment (yosefe) [inline src/ucm/api/ucm.h]: lets rename this to UCM_EVENT_**FLAG**_NO_INSTALL

Comment (yosefe) [inline src/ucm/api/ucm.h]: lets also add to doc of ucm_set_event_handler that using this flag will not trigger installing events but only register the handler

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1349/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1974/ for details.

**Comment by yosefe** [comment]
:+1:

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/1350/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1975/ for details.

**Pull request merged**